src/ios/AppsFlyerPlugin.m

```diff
@@ -35,81 +35,103 @@ - (void)initSdk:(CDVInvokedUrlCommand*)command
         }
-        isDeepLinkingValue = [initSdkOptions objectForKey: afOnDeepLinking];
+
+        isDeepLinkingValue = [initSdkOptions objectForKey:afOnDeepLinking];
+
         if ([isDeepLinkingValue isKindOfClass:[NSNumber class]]) {
             isDeepLinking = [(NSNumber*)isDeepLinkingValue boolValue];
         }
     }
 
     NSString* error = nil;
 
+    // Verify dev key is not null/empty
     if (!devKey || [devKey isEqualToString:@""]) {
         error = NO_DEVKEY_FOUND;
     }
+
     if (!appId || [appId isEqualToString:@""]) {
         error = NO_APPID_FOUND;
     }
 
-    if(error != nil){
-        CDVPluginResult* result = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString: error];
+    // Throw an exception if the error object is not nil
+    if (error != nil){
+        CDVPluginResult* result = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:error];
         [self.commandDelegate sendPluginResult:result callbackId:command.callbackId];
         return;
-    }
-    else{
-        if(isDeepLinking == YES){
+    } else {
+        if (isDeepLinking == YES) {
             [AppsFlyerLib shared].deepLinkDelegate = self;
-         }
+        }
 
-        [[AppsFlyerLib shared] setPluginInfoWith:AFSDKPluginCordova pluginVersion:@"6.12.2" additionalParams:nil];
+        // Initialize the SDK
+        [[AppsFlyerLib shared] setPluginInfoWith:AFSDKPluginCordova pluginVersion:@"6.13.0" additionalParams:nil];
         [AppsFlyerLib shared].appleAppID = appId;
         [AppsFlyerLib shared].appsFlyerDevKey = devKey;
         [AppsFlyerLib shared].isDebug = isDebug;
         [AppsFlyerLib shared].useUninstallSandbox = useUninstallSandbox;
+    }
 
 #ifndef AFSDK_NO_IDFA
-        //Here we set the time that the sdk will wait before he starts the launch. we take the time from the 'option' object in the app's index.js
-        if (@available(iOS 14, *)) {
-            if (waitForATTUserAuthorization != 0 && waitForATTUserAuthorization != nil){
-                [[AppsFlyerLib shared] waitForATTUserAuthorizationWithTimeoutInterval:waitForATTUserAuthorization.intValue];
-                   }
+    // Here we set the time that the SDK will wait before it starts the launch. We take the time from the 'option' object in the app's index.js
+    if (@available(iOS 14, *)) {
+        if (waitForATTUserAuthorization != 0 && waitForATTUserAuthorization != nil) {
+            [[AppsFlyerLib shared] waitForATTUserAuthorizationWithTimeoutInterval:waitForATTUserAuthorization.intValue];
         }
+    }
 #endif
-        [[AppsFlyerLib shared] start];
+    if (![initSdkOptions isKindOfClass:[NSNull class]]) {
+
+        id isConversionDataValue = nil;
+        isConversionDataValue = [initSdkOptions objectForKey:afConversionData];
+
+        if ([isConversionDataValue isKindOfClass:[NSNumber class]]) {
+            isConversionData = [(NSNumber*)isConversionDataValue boolValue];
+        }
+    }
```

Comment:
You already did this check on line 47, so i would move this block of code to the first `if (![initSdkOptions isKindOfClass:[NSNull class]]) {` check

Reply:
done